js/Views.js

```diff
@@ -200,38 +208,35 @@ var Views = (function() {
             console.log("stationId : " + stationId);
 
             // Allow to get distance between station and current position
-            var stations = Stations.filterClosestStations(
-                stationStorage.getStations(),
-                coords
-            );
-            
+            var stations = Stations.filterClosestStations(stationStorage.getStations(), coords);
+
             var station = stationStorage.getStationById(stationId)[0];
             var stationExist = $.grep(stations, function(v) {
                 return v.number == stationId;
             });
 
             // If station doesn't exist : redirection
-            if (stationExist.length == 0) {
+            if (stationExist.length === 0) {
                 alert("La station n'existe pas !");
                 console.log("Views.js", "station", "station doesn't exist", stationExist.length);
                 window.location.hash = "/index";
             } else {
                 currentView = "station";
 
-                console.log('Views', viewStruct.view, "display page");
-                var stationFormatted = Stations.getFormattedStation(station, coords);
-                
+                console.log('Views', currentView, "display page");
+                //var stationFormatted = Stations.getFormattedStation(station, coords);
+
                 var availableBikes = templates['station'].content.querySelector('.bikes');
                 var availableStands = templates['station'].content.querySelector('.stands');
                 var distance = templates['station'].content.querySelector('.distance');
                 var position = templates['station'].content.querySelector('.position');
                 var lastUpdate = templates['station'].content.querySelector('.last_update');
 
-                availableBikes.textContent = stationFormatted.availableBikes;
-                availableStands.textContent = stationFormatted.availableStands;
-                distance.textContent = stationFormatted.distance;
-                position.textContent = stationFormatted.position;
-                lastUpdate.textContent = stationFormatted.lastUpdate;
+                availableBikes.textContent = 0;//stationFormatted.availableBikes;
+                availableStands.textContent = 0;//stationFormatted.availableStands;
+                distance.textContent = 0;//stationFormatted.distance;
+                position.textContent = 0;//stationFormatted.position;
+                lastUpdate.textContent = 0;//stationFormatted.lastUpdate;
```

Comment:
This view was inaccessible because of some changes. 
My goal was just to fix this point. I'll take care of value later.

> Le 11 mars 2015 à 23:50, Elie Michel <notifications@github.com> a écrit :
> 
> In js/Views.js:
> 
> >                  var availableBikes = templates['station'].content.querySelector('.bikes');
> >                  var availableStands = templates['station'].content.querySelector('.stands');
> >                  var distance = templates['station'].content.querySelector('.distance');
> >                  var position = templates['station'].content.querySelector('.position');
> >                  var lastUpdate = templates['station'].content.querySelector('.last_update');
> >  
> > -                availableBikes.textContent = stationFormatted.availableBikes;
> > -                availableStands.textContent = stationFormatted.availableStands;
> > -                distance.textContent = stationFormatted.distance;
> > -                position.textContent = stationFormatted.position;
> > -                lastUpdate.textContent = stationFormatted.lastUpdate;
> > +                availableBikes.textContent = 0;//stationFormatted.availableBikes;
> > +                availableStands.textContent = 0;//stationFormatted.availableStands;
> > +                distance.textContent = 0;//stationFormatted.distance;
> > +                position.textContent = 0;//stationFormatted.position;
> > +                lastUpdate.textContent = 0;//stationFormatted.lastUpdate;
> Forgotten debug, isn't it?
> 
> —
> Reply to this email directly or view it on GitHub.
>